cmake/thirdParties.cmake

```diff
@@ -4,6 +4,9 @@
 
 set(BLA_DEFINITIONS)
 
+set(BLA_VENDOR "OpenBLAS" CACHE STRING "BLAS/LAPACK implementation")
```

Comment:
this overwrites argument passed by user?

Reply:
no it does not (if you want to override you need FORCE)
it set a default value if there were no value

Reply:
if I'm not mistaken, if not set it was complaining to force the user to make a choice.